pontoon/administration/templates/admin.html

```diff
@@ -38,9 +38,9 @@
 {% endblock %}
 
 {% block extend_css %}
-  {% stylesheet 'admin' %}
+  <link href="{{ static('css/admin.min.css') }}" rel="stylesheet" type="text/css">
```

Comment:
Do these changes drop the hashes from the css and js links?

If so, that'd be bad for cached resources in production. Even if we're having a deployment monolith, but more so if we'd go for a CDN.

Reply:
Hmmm, now that you bring this up, the static() method lookup into a map generated by the gulp pipeline but fallback on regular file path without hash when not using gulp, so I may have broken the previous asset serving.
Though I will check if this can be reverted when applying the suggested changes.